data.php

```diff
@@ -0,0 +1,32 @@
+<?php
+require 'functions.php';
+$db = connectToDatabase();
+
+
+if (isset($_POST["name"])) {
+    $name_input = cleanInput($_POST["name"]);
```

Comment:
Worth doing something similar to your rating check for some of these other fields, for example with $name_input I'd check to make sure it's more than say 3 characters after it's been cleaned (just incase a user enters nothing but a space in the form field)